site/pages/our-work/case-study/what-to-read-next/slice.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import InlineSVG from 'svg-inline-react';
+
+import styles from './style.css';
+import arrowSVG from '../../../../../assets/images/SVG/arrow.svg';
+
+type WhatToReadNextSliceProps = {
+  details: Object,
+};
+
+const WhatToReadNextSlice = ({ details }: WhatToReadNextSliceProps) => {
```

Comment:
Here we could extract all the properties out of details so that they are their own properties in the props.

Kinda like this:

```javascript
const WhatToReadNextSlice = ({ name, link, tagline, image }: WhatToReadNextSliceProps) => {
```
This way we can add type-checking for them with flow. And when we use the component we can use the spread operator: `<WhatToReadNextSlice ...details />`
 